src/Monocle/Search/Query.hs

```diff
@@ -162,11 +161,13 @@ queryFieldToDocument name = do
 -- | Resolves the actual document field for a given flavor
 getFlavoredField :: QueryFlavor -> Field -> Maybe Field
 getFlavoredField QueryFlavor {..} field
-  | field `elem` ["author", "group"] = Just $ case qfAuthor of
-    Author -> "author"
-    OnAuthor -> "on_author"
+  | field `elem` ["author", "group"] = Just author
```

Comment:
Yes that's not related ... not the first time I have the issue but the "guards" + case related statements are not formatted the same way with ormolu via vscode and ormolu in ci. Not sure if vscode passes some parameter to ormolu because I only have one ormolu available in the path. Or maybe the hls provides ormolu out of the box ?
So here that's why there is a slight refactor to satisfied ormolu.

Reply:
I think https://github.com/change-metrics/monocle/pull/916 should fix that.